test/unit/transpiler/passes/scheduling/test_scheduler.py

```diff
@@ -845,14 +1028,14 @@ def test_measure_after_c_if(self):
         expected.delay(1000, 2)
         expected.measure(0, 0)
         expected.barrier()
-        expected.x(1).c_if(
-            0, 1
-        )  # Not yet correct as we should insert delays for idle qubits in conditional.
+        with expected.if_test((0, 1)):
+            expected.delay(200, 0)
+            expected.x(1)
+            expected.delay(200, 2)
```

Comment:
This also has the out-of-date comment in the docstring.

Reply:
Fixed in 1e2f0f6.